CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - `ExpandSelection` is now a configurable mouse binding action
 - Config option `background_opacity`, you should use `window.opacity` instead
 - Reload configuration files when their symbolic link is replaced
+- Commands spawn from the current directory of the parent alacritty instance in Unix-like systems
```

Comment:
This comment isn't accurate. They spawn in the current directory of the foreground shell, not in Alacritty's.

Reply:
Corrected.